prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -1,5 +1,8 @@
 <div class="container">
   <div class="row">
+    <div class="col-sm-12">
```

Comment:
Should we allow this to span the entire page?  Might be worth trying something like `row.div.col-sm-12.col-md-9.col-xl-7` with a couple different message lengths.  Also, might be worth encapsulating the responsiveness within the component instead of each use being wrapped so thye're consistent between uses.